packages/strapi-generate-new/lib/index.js

```diff
@@ -36,9 +36,12 @@ module.exports = (projectDirectory, cliArguments) => {
     strapiVersion: require('../package.json').version,
     debug: cliArguments.debug !== undefined,
     quick: cliArguments.quickstart !== undefined,
```

Comment:
you can keep template in the root scope too to keep the code to handle the template more consistent